.ado/publish.yml

```diff
@@ -107,7 +107,7 @@ jobs:
   - job: RNMacOSInitNpmJSPublish
     displayName: react-native-macos-init Publish to npmjs.org
     pool:
-      vmImage: vs2017-win2016
+      vmImage: ubuntu-latest
```

Comment:
I go back and forth on using `latest`. It means builds might jump ahead to an OS version we haven't thoroughly tested which could break our CI pipeline with no changes on our end. Those bugs can get messy and be tricky to attribute the cause to.

Reply:
I'd lean towards picking a version and upgrading it manually whenever we see fit just to keep the CI more under our control

Reply:
We use Ubuntu 18.04 below, so I can keep that. I initially chose latest b/c that's what FURN and RNW use. I'll update